go.mod

```diff
@@ -3,36 +3,9 @@ module github.com/rancher/fleet
 go 1.21
 
 replace (
+	github.com/json-iterator/go v1.1.10 => github.com/json-iterator/go v1.1.12
```

Comment:
Why is this needed? Shouldn't `go get github.com/json-iterator/go@v1.1.12` use that version already?

Reply:
We want to replace the vulnerable version in all transient dependencies. There were a few left in the `go mod graph`. I just moved the directive to the top of the file.

Reply:
Oh, I didn't know about the vulnerability.

Reply:
Removing it like we discussed.